arch/xtensa/core/xtensa-asm2.c

```diff
@@ -202,7 +202,7 @@ void *xtensa_excint1_c(int *interrupted_stack)
 		LOG_ERR(" ** FATAL EXCEPTION");
 		LOG_ERR(" ** CPU %d EXCCAUSE %d (%s)",
 			arch_curr_cpu()->id, cause,
-			z_xtensa_exccause(cause));
+			log_strdup(z_xtensa_exccause(cause)));
```

Comment:
Are you sure this is needed? I thought that `log_strdup()` doesn't have to be used on literal strings, which are saved in ROM. Maybe those strings are not in ROM in case of ESP32?